packages/edit-site/src/components/dataviews/dataviews.js

```diff
@@ -53,14 +58,26 @@ export default function DataViews( {
 		},
 		onSortingChange: ( sortingUpdater ) => {
 			onChangeView( ( currentView ) => {
+				if ( ! sortingUpdater || sortingUpdater.length === 0 ) {
```

Comment:
Is there a case you observed that the updater is not there? I pushed a small update to handle the reset. It would break if we reset sorting by clicking the headers(ex Title)

Reply:
no there's no case that I observed. But we're ok that the last commit was not needed (maybe just a code style difference right?)